src/stores/errors.ts

```diff
@@ -42,8 +24,17 @@ export const schedulingErrors: Writable<SchedulingError[]> = writable([]);
 
 export const allErrors: Readable<BaseError[]> = derived(
   [simulationDatasetErrors, schedulingErrors],
-  ([$simulationDatasetErrors, $schedulingErrors]) => [
-    ...($simulationDatasetErrors ? $simulationDatasetErrors : []),
-    ...($schedulingErrors ? $schedulingErrors : []),
-  ],
+  ([$simulationDatasetErrors, $schedulingErrors]) =>
+    [
+      ...($simulationDatasetErrors ? $simulationDatasetErrors : []),
+      ...($schedulingErrors ? $schedulingErrors : []),
+    ].sort((errorA: BaseError, errorB: BaseError) => {
+      if (errorA.timestamp < errorB.timestamp) {
```

Comment:
Can we use the `compare` function here in utilities?

Reply:
The `compare` function doesn't have the case for when they're equal. Do we care one way or the other? I don't think the result would be that bad.

Reply:
ooo good catch. Maybe we should just add that to compare?